pkg/actions/lua/lakefs/catalogexport/internal.lua

```diff
@@ -59,18 +59,44 @@ end
 
 local function parse_storage_uri(uri)
     local u = url.parse(uri)
-    return { 
+    return {
         protocol = u.scheme,
         bucket = u.host,
         key = (u.path:sub(0, 1) == "/") and u.path:sub(2) or u.path,
     }
 end
 
+local function sortedKeys(query, sortFunction)
+    local keys, len = {}, 0
+    for k,_ in pairs(query) do
+        len = len + 1
+        keys[len] = k
+    end
+    if sortFunction ~= nil then
+        table.sort(keys, sortFunction)
+    else
+        table.sort(keys)
+    end
+
+    return keys
+end
```

Comment:
Why is the sort function needed here, don't see its being used? If it's not needed let's remove it. 
If you prefer to keep it then no need to check for nil: 
 
```suggestion
local function sortedKeys(query, sortFunction)
    local sort = sortFunction or  
    local keys, len = {}, 0
    for k,_ in pairs(query) do
        len = len + 1
        keys[len] = k
    end
    table.sort(keys, sortFunction)
    return keys
end
```

Reply:
It's used in `delta_exporter`.
Not sure I understand the suggested change, but I do agree that the nil check is redundant. 🙏

Reply:
I meant specifically the `sortFunction` argument in the function, I don't see it being used anywhere? 
If you prefer to leave it regardless, please add a comment on the `sortFunction` expected signature otherwise it'll be hard to use it without going into the implementation of `sortedKeys`

Reply:
`sortFunction` is passed to `table.sort` to apply the sorting logic (otherwise it uses default lexicographic sort).
I'll add a comment.